### PR TITLE
change stop button's color to red

### PR DESCRIPTION
**Pull request by ZenoABC**

### CODE REVIEW

File: utils/views.py [hunk 1]
```diff
@@ -248,7 +248,7 @@ def __add_buttons(self) -> None:
 
     def _update_buttons_state(self) -> None:
         button: PaginatorButton
-        for button in [n for n in self.children if n.custom_id != "stop_button"]:  # type: ignore
+        for button in self.children:  # type: ignore
             if button.custom_id in ("page_indicator_button",):
```
Comment (Soheab) [inline utils/views.py]: `"stop_button"` should be in the tuple

**Review by Soheab — CHANGES_REQUESTED** [review]

File: utils/views.py (hunk 1 above)
Comment (ZenoABC) [reply utils/views.py]: done

**Comment by ZenoABC** [comment]
bro i hate this black

**Review by JDJGInc — CHANGES_REQUESTED** [review]
Don't be mean to black

**Pull request merged**